docs/content/flink/cdc-ingestion/pulsar-cdc.md

```diff
@@ -156,6 +160,28 @@ Then you can submit synchronization job:
     ... (other conf)
 ```
 
+Example 3:
+For some append data (such as log data), it can be treated as special CDC data with only INSERT operation type, so you can use 'format=json' to synchronize such data to the Paimon table.
+
+```bash
+<FLINK_HOME>/bin/flink run \
+    /path/to/paimon-flink-action-{{< version >}}.jar \
+    kafka_sync_table \
+    --warehouse hdfs:///path/to/warehouse \
+    --database test_db \
+    --table test_table \
+    --partition_keys pt \
+    --computed_column 'pt=date_format(event_tm, yyyyMMdd)' \
+    --kafka_conf properties.bootstrap.servers=127.0.0.1:9020 \
+    --kafka_conf topic=test_log \
+    --kafka_conf properties.group.id=123456 \
+    --kafka_conf value.format=json \
+    --catalog_conf metastore=hive \
+    --catalog_conf uri=thrift://hive-metastore:9083 \
+    --table_conf bucket=4 \
```

Comment:
really work? you should no config, use default value.

Reply:
sorry, my oversight